lib/Game.js

```diff
@@ -15,7 +14,11 @@ class Game {
   }
 
   changePlayerTurn() {
```

Comment:
Based on peer review - do we want to only change player when they answer the clue incorrectly?

Reply:
Reviewed together - we will stick with project specs.